vlib/regex/README.md

```diff
@@ -90,8 +90,8 @@ For example `\w` is the meta-char `w`.
 
 A meta-char can match different types of characters.
 
-* `\w` matches an alphanumeric char `[a-zA-Z0-9_]`
-* `\W` matches a non alphanumeric char
+* `\w` matches an word char char `[a-zA-Z0-9_]`
```

Comment:
**a** word char  :-)

Reply:
It's late here, I'm tired and you are right, tnx for the note :)